packages/core/app-layout/src/components/aboutSection/AppAboutSection.vue

```diff
@@ -4,14 +4,18 @@
     title-tag="h2"
   >
     <template
-      v-if="title"
+      v-if="$slots.title || title"
       #title
     >
       <span
         class="about-section-title"
         data-testid="about-section-title"
       >
-        {{ title }}
+        <slot
+          v-if="$slots.title"
+          name="title"
+        />
+        <template v-else>{{ title }}</template>
```

Comment:
This can be simplified by utilizing the prop value as the default slot content.
```suggestion
        <slot name="title">
          {{ title }}
        </slot>
```

Reply:
Yes, you are right!